wiki/Beatmap_Discussion/en.md

```diff
@@ -171,21 +196,17 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
 
 ## FAQ
 
-### Is Kudosu! available in Beatmap Discussion?
-
-Kudosu! has been recently implemented in the Beatmap Discussion! You can give modders Kudosu! by using the thumbs up emote on discussions that were marked as Suggestions or Problems.
-
-### How are beatmaps nominated?
+### How do I edit my beatmap's description with Beatmap Discussion?
 
-Beatmap Discussion follows the same procedure as its predecessor. 2 Beatmap Nominators are required to nominate the Meatmapset for qualification. Star Priority has been replaced with Hype and requires 12 Hype before it can be nominated.
+You can edit your beatmap's description on the new site by either submitting your map and editing it like in the past, or you may simply edit it at any time by clicking on the edit icon in the top right of the description field in the beatmapset's page.
```

Comment:
bit redundant since it's the same thing a couple lines up

Reply:
removed, I'm running out of time, so I'll have good faith and assume so